scripts/python/osinstall.py

```diff
@@ -45,7 +47,7 @@
 IPR = IPRoute()
 
 PROFILE = os.path.join(GEN_PATH, 'profile.yml')
-NODE_LIST = os.path.join(GEN_PATH, 'node_list.yml')
+NODE_STATUS = os.path.join(GEN_PATH, 'osinstall_node_status.yml')
 
 HTTP_ROOT_DIR = '/srv'
```

Comment:
It doesn't need to be with this commit, but the setting of pup's http root should probably be in genesis.py so that the software installer and osinstall are getting the same info

Reply:
Fixed in 0c2d14e